packages/block-editor/src/components/block-patterns-list/style.scss

```diff
@@ -18,8 +18,9 @@
 		display: flex;
 		align-items: center;
 		overflow: hidden;
-		border-radius: $radius-block-ui + $border-width;
+		border-radius: 4px;
 		border: $border-width solid $gray-100;
+		box-shadow: 0 15px 25px rgb(0 0 0 / 7%);
```

Comment:
Should we make this a var [like we did with modal/popover shadows](https://github.com/WordPress/gutenberg/blob/3a2a5559b60540ae2098b633f9b54d7676a61ba5/packages/base-styles/_variables.scss#L66-L67)? There may be other places in the UI where we want to use the same effect.

Reply:
I'd love some system for shadows, a pet peeve of mine is to "obey the laws of physics". This is a very soft shadow, suggesting anything above it should be even softer, i.e. modals.

Reply:
There is an [experimental elevation](https://wordpress.github.io/gutenberg/?path=/story/components-experimental-elevation--default) component, but I don't know if it's used anywhere.